src/app/beer_garden/api/http/handlers/v1/job.py

```diff
@@ -210,11 +217,145 @@ async def post(self):
             Operation(
                 operation_type="JOB_CREATE",
                 args=[
-                    SchemaParser.parse_job(self.request.decoded_body, from_string=True)
+                    SchemaParser.parse_job(
+                        self.request.body.decode("utf-8"), from_string=True
+                    )
                 ],
             )
         )
 
         self.set_status(201)
         self.set_header("Content-Type", "application/json; charset=UTF-8")
         self.write(response)
+
+
+class JobImportAPI(BaseHandler):
+    def data_received(self, chunk: bytes) -> Optional[Awaitable[None]]:
+        return super().data_received(chunk)
+
+    @authenticated(permissions=[Permissions.CREATE])
+    async def post(self):
+        """
+        ---
+        summary: Schedule a list of Jobs from a list of job descriptions.
+        description: |
+          Given a list of jobs from /export/jobs, each will be scheduled to run
+          on the intervals that are set in their trigger arguments.
+        parameters:
+          - name: jobs
+            in: body
+            description: The Jobs to create/schedule
+            schema:
+              $ref: '#/definitions/JobExport'
+        responses:
+          201:
+            description: All new jobs have been created
+            schema:
+              $ref: '#/definitions/JobImport'
+          400:
+            $ref: '#/definitions/400Error'
+          50x:
+            $ref: '#/definitions/50xError'
+        tags:
+          - Jobs
+        """
+        empty_body_regex = re.compile(r"\s*\{\s*\}\s*")  # noqa
+        decoded_body: str = self.request.body.decode("utf-8")
+
+        if not len(decoded_body) or empty_body_regex.match(decoded_body):
+            raise ValueError("Body was empty")
+
+        parsed_job_dfn_list: Optional[JobDefinitionList]
+        parsed_job_dfn_list = SchemaParser.parse_job_definitions(
+            decoded_body, from_string=True
+        )
+
+        if parsed_job_dfn_list is None or parsed_job_dfn_list.jobs is None:
+            raise ValueError("Could not parse any job definitions")
+
+        cleaned_job_dfn_list = JobDefinitionList(
+            list(map(self._clean_job, parsed_job_dfn_list.jobs))
+        )
+
+        response = await self.client(
+            Operation(operation_type="JOB_CREATE_MULTI", args=[cleaned_job_dfn_list])
+        )
+
+        self.set_status(201)
+        self.set_header("Content-Type", "application/json; charset=UTF-8")
+        self.write(response)
+
+    @classmethod
+    def _clean_job(cls, job: brewtils.models.Job) -> brewtils.models.Job:
```

Comment:
I made a related comment on the brewtils side, but I think rather than doing this whole cleanup thing, you can just use a schema that is specific to the import and export process which doesn't include this fields.

Reply:
Done.